api/python/provisioner/commands/configure_setup.py

```diff
@@ -351,7 +351,7 @@ def _key_int_to_list(self, contents):
     def _parse_input(self, input):
         kv_to_dict = dict()
         for key in input.keys():
-            if "," in input.get(key):
+            if input.get(key) and "," in input.get(key):
```

Comment:
it would work, I just would suggest:
- do not use `input` since it is one of built-in functions
- iterate using `for k, v in input.items()` to avoid calling get multiple times